File: code/java/twoSumII.java
class Solution {
    public int[] twoSumII(int[] numbers, int target) {
        int i = 0;
        int j = numbers.length - 1;
        int[] res = new int[2];
        while(i != j)
            if(numbers[i]+numbers[j] == target){
                res[0] = ++i;
                res[1] = ++j;
                return res;
            }
            else if(numbers[i]+numbers[j] < target)
                i++;
            else
                j--;
        return res;
    }
}

File: code/python/twoSumII.py
class Solution:
    def twoSum(self, numbers: List[int], target: int) -> List[int]:
        i = 0;
        j = len(numbers) - 1
        while i != j:
            if numbers[i] + numbers[j] == target:
                return list([i+1,j+1])
            elif numbers[i] + numbers[j] < target:
                i = i + 1
            else:
                j = j - 1
